File: problem-1/sumbyto.hs
sumbyto :: Int -> Int -> Int -> Int
sumbyto start by to
    | start > to    = 0
    | otherwise = start + sumbyto (start + by) by to

main = do
    print (sumbyto 0 3 999 + sumbyto 0 5 999 - sumbyto 0 15 999)

File: problem-1/sumbyto.jl
function sumbyto(start, by, to)
	if start > to
		return 0
	end

	return start + sumbyto(start + by, by, to)
end

mult_a = parse(Int, ARGS[1])
mult_b = parse(Int, ARGS[2])
max = parse(Int, ARGS[3])

println(sumbyto(0, mult_a, max) + sumbyto(0, mult_b, max) - sumbyto(0, 15, max))

